ol-cli/src/machine.rs

```diff
@@ -0,0 +1,155 @@
+//! `machine` 
+// pub mod combined_state_machine {
+//     use sm::Machine;
+//     use sm::Transition;
+//     use super::node_state_machine::export;
+//     // use super::node_state_machine::NodeState::RunWizard;
+
+//     pub fn test() {
+//         let mac = export();
+//         // let mac = mac.transition(RunWizard);
+//         dbg!(&mac.state());
+//     }
+ 
+
+//         // let sm = Machine::new(StateEmptyBox);
+//         // println!("state: {:?} (trigger: {:?})", sm.state(), sm.trigger());
+    
+//         // let sm = sm.transition(RunWizard);
+//         // println!("state: {:?} (trigger: {:?})", sm.state(), sm.trigger().unwrap());
+// }
+// A state machine description for validator node onboarding
+
+// pub mod node_state_machine {
```

Comment:
I entirely understand what this commented out bit is for? is it supposed to tell me something about `sm!` macro?